docs/what-are-product-design-sprints.md

```diff
@@ -0,0 +1,34 @@
+---
+id: product-design-sprint
+title: What are Product Design Sprints?
+sidebar_label: What are Product Design Sprints?
+---
+
+All our projects are design-led, starting with post-its and sketching, ending
+with design implemented as code in the app. We use design sprints and user
+research to understand our client's problems, validate assumptions about the
+products, and build products that are user-centered.
+
+## Introduction to Product Design Sprints
+
+> "Most people make the mistake of thinking design is what it looks like.
+> People think it's this veneer — that the designers are handed this box and
+> told, 'Make it look good!' That's not what we think design is. It's not just
+> what it looks like and feels like. Design is how it works." - Steve Jobs
+
+[Product Design
+Sprints](https://robots.thoughtbot.com/the-product-design-sprint) an invention
+of [Google Ventures' Design Team](http://www.gv.com/team/#design), are 5-phase
+exercises intended to improve the chances of [making something people
+want](http://paulgraham.com/good.html). We want to turn false confidence into
+validated confidence before beginning an expensive build. Or, we want to dodge
+bullets by learning we shouldn't begin the expensive build at all.
+
+Sprints are useful starting points when kicking off a new product or workflow,
+as well as solving problems with an existing product. They typically last 5 days
```

Comment:
awkwardly phrased.

Reply:
How would you phrase it?

Reply:
"They typically last 5 days but we have done them in less time because we try to get as many stakeholders and expertises in the room as we can."